### PR TITLE
Keep pre commit & push passing

### PR DESCRIPTION
**Pull request by KPrasch**

### CODE REVIEW

**Comment by jMyles** [comment]
Looks good, but what's the change to the registry?

**Review by cygnusv — CHANGES_REQUESTED** [review]

**Comment by KPrasch** [comment]
This is an odd PR, I might even close it, or at least try to alter the configuration so that some of the effected files are ignored.

With regard to the registry - do we want the newline?  The diff shows it as a massive change, when in fact it is only one character.

**Comment by derekpierre** [comment]
@KPrasch we can use the `exclude` option (https://pre-commit.com/#pre-commit-configyaml---top-level) to ignore some files.

You can probably start a new PR, or just alter the configuration and do a force push.

**Comment by KPrasch** [comment]
@derekpierre Perfect.  

That's seems like a good outcome to me.

Consider this PR just a demonstration.

File: nucypher/blockchain/eth/contract_registry/mainnet/contract_registry.json [hunk 1]
```diff
@@ -1 +1 @@
...
...ss"}], "name": "transferOwnership", "outputs": [], "stateMutability": "nonpayable", "type": "function"}, {"inputs": [{"internalType": "uint256", "name": "_gasToSaveState", "type": "uint256"}], "name": "verifyBiddingCorrectness", "outputs": [{"internalType": "uint256", "name": "", "type": "uint256"}], "stateMutability": "nonpayable", "type": "function"}, {"inputs": [], "name": "withdrawCompensation", "outputs": [], "stateMutability": "nonpayable", "type": "function"}, {"inputs": [{"internalType": "address", "name": "", "type": "address"}], "name": "workInfo", "outputs": [{"internalType": "uint256", "name": "depositedETH", "type": "uint256"}, {"internalType": "uint256", "name": "completedWork", "type": "uint256"}, {"internalType": "bool", "name": "claimed", "type": "bool"}, {"internalType": "uint128", "name": "index", "type": "uint128"}], "stateMutability": "view", "type": "function"}, {"inputs": [{"internalType": "uint256", "name": "_completedWork", "type": "uint256"}, {"internalType": "uint256", "name": "_depositedETH", "type": "uint256"}], "name": "workToETH", "outputs": [{"internalType": "uint256", "name": "", "type": "uint256"}], "stateMutability": "view", "type": "function"}]]]
```
[the remark this answers is not in the document]
Comment (KPrasch) [reply nucypher/blockchain/eth/contract_registry/mainnet/contract_registry.json:1]: Yes - I agree.

[the remark this answers is not in the document]
Comment (KPrasch) [reply nucypher/blockchain/eth/contract_registry/mainnet/contract_registry.json]: :heavy_check_mark:

**Comment by KPrasch** [comment]
Alright - PR is no longer a demo - @derekpierre and I adjusted the pre commit config to ignore all the registries.

**Review by derekpierre — APPROVED** [review]
🎸

**Review by cygnusv — APPROVED** [review]

**Pull request merged**